yt_dlp/extractor/zoom.py

```diff
@@ -51,27 +47,48 @@ def _real_extract(self, url):
                 raise ExtractorError(validation['errorMessage'], expected=True)
             webpage = self._download_webpage(url, play_id)
 
+        data: dict[str, Union[None, str, int, list, bool, dict[str, Any]]]
```

Comment:
```suggestion
```

Reply:
why shouldn't I use typing declarations?
I'm pretty sure its compatible with the python version yt-dlp supports

Reply:
They are not needed. The next dev working on this extractor may get confused as to why typing is used specifically for this extractor.

Reply:
> why shouldn't I use typing declarations?

simply because I don't want to deal with them